# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: gfx/wr/webrender_api/src/api.rs
# Commit: 5506d1803e15
# Full Hash: 5506d1803e158df5504bc74a7818cb78dca6420c
# Author: Kris Taeleman <ktaeleman@mozilla.com>
# Date: 2019-10-16 05:20:00
# Description:
#   Bug 1585801 - Fixing crash when folder is not writable and refactored logic. r=jrmuizel
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D48999
# ==============================================================================

diff -r f24ddb55b48b -r 5506d1803e15 gfx/wr/webrender_api/src/api.rs
--- a/gfx/wr/webrender_api/src/api.rs	Sat Oct 12 23:50:45 2019 +0000
+++ b/gfx/wr/webrender_api/src/api.rs	Tue Oct 15 17:03:22 2019 +0000
@@ -775,6 +775,8 @@
     /// Causes the low priority scene builder to pause for a given amount of milliseconds
     /// each time it processes a transaction.
     SimulateLongLowPrioritySceneBuild(u32),
+    // Logs transactions to a file for debugging purposes
+    SetTransactionLogging(bool),
 }
 
 #[derive(Clone, Deserialize, Serialize)]
@@ -1140,7 +1142,6 @@
         const DISABLE_TEXT_PRIMS = 1 << 22;
         const DISABLE_GRADIENT_PRIMS = 1 << 23;
         const OBSCURE_IMAGES = 1 << 24;
-        const LOG_TRANSACTIONS = 1 << 25;
     }
 }
 